src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -570,13 +570,8 @@
 .#{$menu-toggle}__icon {
   flex-shrink: 0;
 
-  &.pf-m-avatar {
-    .#{$avatar},
-    img,
-    svg {
-      margin-block-start: calc(var(--#{$menu-toggle}--PaddingBlockStart) * -1);
-      margin-block-end: calc(var(--#{$menu-toggle}--PaddingBlockEnd) * -1);
-    }
```

Comment:
We'll want to keep this to avoid a breaking change, but I think taking it out of the docs like you did is great, and we'll also want to take `menu-toggle-icon--HasAvatar` off of these examples

https://github.com/patternfly/patternfly/blob/cbe70fe435e751db884c38bf1b9bfac53ef4bab7/src/patternfly/components/MenuToggle/examples/MenuToggle.md?plain=1#L283

https://github.com/patternfly/patternfly/blob/cbe70fe435e751db884c38bf1b9bfac53ef4bab7/src/patternfly/components/MenuToggle/examples/MenuToggle.md?plain=1#L297

https://github.com/patternfly/patternfly/blob/cbe70fe435e751db884c38bf1b9bfac53ef4bab7/src/patternfly/components/MenuToggle/examples/MenuToggle.md?plain=1#L311